src/plugins/data_views/server/fetcher/lib/field_capabilities/field_caps_response.ts

```diff
@@ -133,6 +142,12 @@ export function readFieldCapsResponse(
         aggregatable: isAggregatable,
         readFromDocValues: shouldReadFieldFromDocValues(isAggregatable, esType),
         metadata_field: capsByType[types[0]].metadata_field,
+        meta: {
+          fixedInterval: capsByType[types[0]].meta?.fixed_interval,
+          timeZone: capsByType[types[0]].meta?.time_zone,
+        },
+        timeSeriesMetric: timeSeriesMetricType,
```

Comment:
Should we add https://github.com/elastic/kibana/pull/136833/files#diff-7fd8f81570bb34b0ee8771f29f9ace6925c5cfa7144f75ede8bdc09275160f19R142 to this, too? This makes it possible to check for rollup fields very specifically.

Reply:
one can inspect the es types, but i agree this might be a nice but we can probably just implement it on the Field instance rather than adding it to spec and field caps response.